### PR TITLE
highfive: bump to v2.2.1

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Review by matz-e — DISMISSED** [review]
Sure! For consistency, shouldn't it have been `2.2.0`?

**Comment by alkino** [comment]
> Sure! For consistency, shouldn't it have been `2.2.0`?

Version are with two numbers, we add a third number if we need a hot fix

File: var/spack/repos/builtin/packages/highfive/package.py [hunk 1]
```diff
@@ -45,10 +46,16 @@ def cmake_args(self):
...
+            '-DHIGHFIVE_EXAMPLES:Bool='
+            + str(self.spec.satisfies('@develop')),
+            '-DHIGHFIVE_UNIT_TESTS:Bool='
+            + str(self.spec.satisfies('@develop')),
+            '-DHIGHFIVE_TEST_SINGLE_INCLUDES:Bool='
+            + str(self.spec.satisfies('@develop')),
             '-DHDF5_NO_FIND_PACKAGE_CONFIG_FILE=1',  # Dont use targets
-            #'-DHIGHFIVE_USE_INSTALL_DEPS:Bool=ON',  # Newer highfive. Otherwise dynamic deps
+
+            # Newer highfive. Otherwise dynamic deps
```
Comment (ferdonline) [inline var/spack/repos/builtin/packages/highfive/package.py]: Isn’t it the time to uncomment this?

**Comment by alkino** [comment]
Please retest

**Comment by alkino** [comment]
Please retest

**Review by matz-e — DISMISSED** [review]

**Comment by matz-e** [comment]
Retest this please

**Review by matz-e — APPROVED** [review]

**Comment by alkino** [comment]
Hurray o/

**Pull request merged**